crates/trycp_client/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "trycp_client"
+version = "0.0.100-alpha0"
+license = "CAL-1.0"
+edition = "2021"
+
+[dependencies]
+futures = "0.3.30"
+rmp-serde = "1.1"
+serde_json = "1.0.117"
+tokio = { version = "1.38.0", features = [ "full" ] }
+tokio-tungstenite = "0.21"
+trycp_api = { version = "0.0.100-alpha0", path = "../trycp_api" }
```

Comment:
To the workspace for easy finding when updating versions? No release-automation here :)